backend/program/downloaders/realdebrid.py

```diff
@@ -63,7 +63,11 @@ def validate(self) -> bool:
         try:
             response = ping(f"{RD_BASE_URL}/user", additional_headers=self.auth_headers, proxies=self.proxy)
             if response.ok:
-                user_info = response.json()
+                try:
+                    user_info = response.json()
+                except:
+                    logger.error("Couldn't parse user data response from Real-Debrid.")
```

Comment:
do you have an example of when this happens?

Reply:
If rd has maintenance (15 minutes ago). I got exceptions because it returned html instead of valid json

Reply:
ah okay, perfect. Good catch! It might be worth just to wrap the response into the try-except as well

Reply:
Lets fix this so that we're not "double wrapping" with 2 try-except's.

Reply:
I mean there is one already, but it didn't catch it correctly..

Reply:
dang